test/parallel/test-stdio-closed.js

```diff
@@ -2,28 +2,23 @@
 const common = require('../common');
 const assert = require('assert');
 const spawn = require('child_process').spawn;
+const fs = require('fs');
 
 if (common.isWindows) {
   common.skip('platform not supported.');
   return;
 }
 
 if (process.argv[2] === 'child') {
-  try {
-    process.stdout.write('stdout', function() {
-      try {
-        process.stderr.write('stderr', function() {
-          process.exit(42);
-        });
-      } catch (e) {
-        process.exit(84);
-      }
-    });
-  } catch (e) {
-    assert.strictEqual(e.code, 'EBADF');
-    assert.strictEqual(e.message, 'EBADF: bad file descriptor, write');
-    process.exit(126);
-  }
+  assert.doesNotThrow(function() {
+    fs.fstatSync(0)
+  });
+  assert.doesNotThrow(function() {
+    fs.fstatSync(1)
+  });
+  assert.doesNotThrow(function() {
+    fs.fstatSync(2)
+  });
```

Comment:
Extremely minor nit... this can be much more succinctly written...

```js
[0,1,2].forEach((i) => assert.doesNotThrow(() => fs.fstatSync(i)));
```

But not necessary to land :-)